configuration/pih/pih-config-haiti-belladere.json

```diff
@@ -36,9 +36,7 @@
   },
 
   "locationTags": {
-    "Login Location": ["Belladère"],
-    "Medical Record Location": ["Belladère"],
-    "Visit Location": ["Belladère"]
```

Comment:
It's possible that Medical Record Location and Visit Location are two examples of ones that need to configured on a site-by-site basis, like Login, but lets go for moving them and see what happens.

Once moved, would be good to confirm that you can't start a visit at an unexpected location or create/request a paper record